lua/ulx/modules/sh/cfc_constraint_checker.lua

```diff
@@ -128,42 +128,47 @@ local function compileConstraintResults( _, ply, constraintCounts )
     return blockData
 end
 
+local function getMsgCArgs( constraintData )
+    local args = {}
+    for _, data in pairs( constraintData ) do
+        table.Add(args, compileConstraintResults( data.ply, data.counts ))
+    end
+
+    return args
+end
 
 function cmd.checkConstraints( caller, targetPlys, showPlysWithNoConstraints )
     local perPlyConstraints = countConstraints( targetPlys )
-    local dataBlocks = {}
 
     ulx.fancyLogAdmin( caller, true, "#A checked the constraints of #T", targetPlys ) -- Alert staff console of the command being used
 
+    -- Convert constraint data to list and remove players with no constraints if necessary
+    local constraintCountsList = {}
     for _, ply in pairs( targetPlys ) do
-        local constraintCounts = perPlyConstraints[ply]
-        if showPlysWithNoConstraints or constraintCounts.Total > 0 then
-            local block = compileConstraintResults( caller, ply, constraintCounts )
-            table.insert( dataBlocks, block )
+        local plyCounts = perPlyConstraints[ply]
+        if showPlysWithNoConstraints or plyCounts.Total > 0 then
+            table.insert( constraintCountsList, {
```

Comment:
modify this code to convert data to a list so its easier to modify and sort before converting to MsgC args.